apis/iam/v1alpha1/policy_identity.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+// PolicyIdentity defines the resource reference to IAMDenyPolicy, which "External" field
+// holds the GCP identifier for the KRM object.
+type PolicyIdentity struct {
+	parent *PolicyParent
+	id     string
+}
+
+func (i *PolicyIdentity) String() string {
+	return i.parent.String() + "/policys/" + i.id
```

Comment:
nit:

```suggestion
	return i.parent.String() + "/policies/" + i.id
```

although we know this doesn't get tested till later 😛